clang/runtime/dpct-rt/include/math.hpp

```diff
@@ -547,6 +585,122 @@ inline unsigned vectorized_sum_abs_diff(unsigned a, unsigned b) {
   }
   return sum;
 }
+
+template <typename RetT, typename AT, typename BT>
```

Comment:
add simple comment for those public math API.

Reply:
done